src/lib/swapper/swappers/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -195,9 +195,15 @@ export async function getTradeQuote(
           .dividedBy(bn(selectedLifiRoute.fromAmount))
           .toString()
 
+        const estimatedExecutionTimeMs = selectedLifiRoute.steps.reduce(
+          (acc, step) => acc + 1000 * step.estimate.executionDuration,
```

Comment:
Confirmed this reduce bit is how Jumper are doing estimations as well 🐐 
https://github.com/lifinance/widget/blob/9b8afd034adab81e74e62d9b484446086d2b19fa/packages/widget/src/components/RouteCard/RouteCardEssentials.tsx#L16-L19C54